CHANGES

```diff
@@ -17,6 +17,7 @@ Bugs fixed
 * latex: allow one more nesting depth for code-blocks (ref #777, PR #2624)
 * #2635: Latex code directives produce inconsistent frames based on viewing resolution
 * #2639: Sphinx now bundles iftex.sty
+* Sphinx now bundles framed.sty
```

Comment:
@tk0miya shouldn't this line be suppressed if we use \OuterFrameSep only conditionally? no need to ship framed.sty then. Memo: version in TeXLive 2009 of framed.sty is ``v 0.95    2007/10/04``, the one current is ``v 0.96    2011/10/22`` hence shipped with TL-2012, or up-to-date TL-2011.